cmd/experimental/kjobctl/pkg/cmd/create/create.go

```diff
@@ -288,7 +303,8 @@ var createModeSubcommands = map[string]modeSubcommand{
 			subcmd.Use += " [--cmd COMMAND]" +
 				" [--replicas [WORKER_GROUP]=REPLICAS]" +
 				" [--min-replicas [WORKER_GROUP]=MIN_REPLICAS]" +
-				" [--max-replicas [WORKER_GROUP]=MAX_REPLICAS]"
+				" [--min-replicas [WORKER_GROUP]=MIN_REPLICAS]" +
```

Comment:
max replicas is gone and min replicas is duplicated

Reply:
Thanks!